instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/servlet/ServletContextPath.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.servlet;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.ContextKey;
+
+public class ServletContextPath {
```

Comment:
ServletContextContextUtils? Just kidding :)

Reply:
Let's add some class javadoc to explain what's going on, including why this needs to go in instrumentation-api

Reply:
👍 will do tomorrow before merging